packages/sdk/contractkit/src/wrappers/FeeCurrencyDirectoryWrapper.test.ts

```diff
@@ -0,0 +1,13 @@
+import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { newKitFromWeb3 } from '../kit'
+
+testWithGanache('FeeCurrencyDirectory', (web3) => {
+  const kit = newKitFromWeb3(web3)
+
+  // Fails with "FeeCurrencyDirectory not (yet) registered"
+  it.failing('fetches fee currency information', async () => {
```

Comment:
this could be not failing now if updated to use testwithAnvil

Reply:
done in [this commit](https://github.com/celo-org/developer-tooling/pull/236/commits/9d47b219979ec3cfdb60110450a6164941276578)